.github/workflows/ci-model-regression-on-schedule.yml

```diff
@@ -0,0 +1,225 @@
+name: CI - Model Regression
+
+on:
+  schedule:
+    - cron:  '1 23 * * */7'
+
+env:
+  GKE_ZONE: us-central1-a
+
+jobs:
+  read_test_configuration:
+    name: Reads tests configuration
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+    - name: Checkout master
+      uses: actions/checkout@v2
+
+    - name: Download yq
+      run: |
+        curl --location https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o yq
+        chmod +x yq
+
+    - run: test -f .github/configs/mr-test-schedule.yaml
```

Comment:
can we please add a name to this step? I was a bit confused where the `test` is coming from

Reply:
same for the other steps.